docs/DeveloperGuide.md

```diff
@@ -155,6 +155,56 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### View Person Feature
+
+#### Implementation
+
+To make the GUI cleaner and simpler, we decided not to display the schedule of each person in the person list.
+Instead, we made another panel on the right that will display the schedule of each person upon calling the function View.
+
+To do this, we made a new `FilteredList<Person>` object called `ViewedPersonList` in `ModelManager` to store the persons that we want to show.
+Moreover, we created `PersonViewPanel.java`, `PersonViewCard.java`, and their respective `.fxml` files so it will be 
+easier to maintain or develop.
+
+Overall, how this command works is similar to a combination of `delete` and `find`, in which we only take a particular index
+as input, and we filter the `ViewPersonList` by some criteria.
```

Comment:
```suggestion
Overall, how this command works is similar to a combination of `find` and `delete`, in which we only take an index as input, and we retrieve information based on the filtered list.
```
I think we should mention it based on the order we call the command. Also, filter by some criteria is not very clear.